tests/unit/strict-resolver/basic-test.js

```diff
@@ -135,3 +137,148 @@ test("can lookup a view in another namespace with different syntax", function(as
   assert.ok(view, 'view was returned');
   assert.equal(view, expected, 'default export was returned');
 });
+
+test("can lookup a view", function(assert) {
```

Comment:
Mostly just adding tests from ember-resolver as a starting point. Getting to the point where each new test brings up design questions.